src/test/unit/java/godaddy/registry/jtoolkit2/TestEnvironment.java

```diff
@@ -236,4 +236,14 @@ public void setClientID(String id) {
     public boolean enforceStrictValidation() {
         return true;
     }
+
+    @Override
+    public byte[] getKeyStoreFileBytes() {
+        return null;
+    }
+
+    @Override
+    public byte[] getTrustStoreFileBytes() {
+        return null;
+    }
```

Comment:
It does not look like we need these two methods, please refer to our comment in `SessionProperties.java`.

Reply:
Same reason

Reply:
Please see my reply in `SessionProperties`.

Reply:
Removed two methods